kroxylicious-docs/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
+        <version>0.13.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>kroxylicious-docs</artifactId>
+    <description>Documentation for Kroxylicious</description>
```

Comment:
If we're make the docs fit into the maven build, what is their `<packaging>`?

Reply:
You mean something like:

```
<packaging>tgz</packaging>
```

https://www.baeldung.com/maven-packaging-types talks about 'Other packaging types' by way of custom plugins but I don't think I've ever seen this in the wild.  In the past I have used the maven-assembly-plugin to produce additional artefacts.